### ISSUE TITLE
Regarding the HuberLoss parameter

### ISSUE THREAD

**Issue by LucasHyun** [rdh1115/T-Graphormer#1]
Hi, I hope you're doing well. I read your paper well and tried to replicate the experiment.
And I found that initially, Huberloss is causing some problems for our code. So I changed it to 
"nn.Huberloss(reduction='mean', delta = args.loss_delta)" from "nn.Huberloss(args.loss_delta)"
I assumed that the reduction is 'none'. 
Could you please confirm which reduction you used for the experiment?

**Comment by rdh1115** [comment]
Hi, I hope you're doing well too. 

I used the default reduction for HuberLoss [[source]](https://docs.pytorch.org/docs/stable/_modules/torch/nn/modules/loss.html#HuberLoss):
``` Python
torch.nn.HuberLoss(reduction='mean')
```

Thanks for reading the paper! Let me know if you have any further questions.
